Bottom: 42d85c6966b786dfb9deeb41eeb8bf86829b8dfb
Top:    703a56e8474034da68b9e4fcf56df2a9eddbe50b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 22:22:10 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 6bb896a..2e7250d 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -589,8 +589,6 @@ ghid_use_mask (int use_it)
 {
   static int stencil_bit = 0;
 
-#warning FIX THIS UP TO COOPORATE WITH HIDGL'S ASSIGNMENT OF STENCIL BITPLANES
-
   /* THE FOLLOWING IS COMPLETE ABUSE OF THIS MASK RENDERING API... NOT IMPLEMENTED */
   if (use_it == HID_LIVE_DRAWING ||
       use_it == HID_LIVE_DRAWING_OFF ||
